<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveAndRestoreOptionsPage</class>
 <widget class="QWidget" name="SaveAndRestoreOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="saveStateLabel">
     <property name="text">
      <string>XenAdmin Console can save and restore your connected servers and their passwords.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="saveStateCheckBox">
     <property name="text">
      <string>Save sessions on exit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="savePasswordsCheckBox">
     <property name="text">
      <string>Save passwords</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoReconnectCheckBox">
     <property name="text">
      <string>Auto-reconnect on startup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mainPasswordGroupBox">
     <property name="title">
      <string>Main Password</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="mainPasswordLabel">
        <property name="text">
         <string>You can set a main password to protect your saved server credentials.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="requireMainPasswordCheckBox">
          <property name="text">
           <string>Require main password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeMainPasswordButton">
          <property name="text">
           <string>Change Password</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="cryptoUnavailableLabel">
        <property name="text">
         <string>This feature is not available because the app was compiled without crypto support.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
